src/components/common/TouchableTextIcon.js

```diff
@@ -0,0 +1,48 @@
+// @flow
+
+import * as React from 'react'
+import { StyleSheet, TouchableOpacity } from 'react-native'
+import MaterialIcons from 'react-native-vector-icons/MaterialIcons'
+
+import T from '../../modules/UI/components/FormattedText/FormattedText.ui.js'
+import { THEME } from '../../theme/variables/airbitz.js'
+import { scale } from '../../util/scaling.js'
+
+type Props = {
+  containerStyle?: StyleSheet.Styles,
```

Comment:
Never take style props like this. If we need to customize the component, add explicit props for the things that need changing.